maximbircu/devtools-library
# Implement a tool context menu with help and state reset actions

Closes: #36 


### What has been done
 1. Implement a context menu for all dev tools with;
 2. Implement a Help dialog which displays:
- Tool title;
- Tool description;
- Tool key;
- Tool current configuration value;
- Tool default configuration value.
3. Fixed number exception crash which happens when passing empty values for number text tools.

### Screenshots
Android Context Menu | Android Help Dialog | iOS
:-: | :-: | :-:
<img src="https://user-images.githubusercontent.com/12527390/80267900-29159a80-86ac-11ea-9ef3-d5c172148a3a.png" width=300/>|<img src="https://user-images.githubusercontent.com/12527390/80267925-4480a580-86ac-11ea-86e9-c23f68c164e2.png" width=300/>| iOS view implementations will be added in|

### How to test
1. Open the sample app;
1. Select any source;
1. Notice dev tool items have a context menu, make sure all context menu actions are working properly;
1. Make sure the Help Dialog shows proper data.
